event/routability.go

```diff
@@ -1,21 +1,13 @@
 package event
 
-// EvtLocalRoutabilityPrivate is an event struct to be emitted with the local's
-// node routability changes to PRIVATE (i.e. not routable from the Internet).
-//
-// This event is usually emitted by the AutoNAT subsystem.
-type EvtLocalRoutabilityPrivate struct{}
-
-// EvtLocalRoutabilityPublic is an event struct to be emitted with the local's
-// node routability changes to PUBLIC (i.e. appear to routable from the
-// Internet).
-//
-// This event is usually emitted by the AutoNAT subsystem.
-type EvtLocalRoutabilityPublic struct{}
+import (
+	"github.com/libp2p/go-libp2p-core/network"
+)
 
-// EvtLocalRoutabilityUnknown is an event struct to be emitted with the local's
-// node routability changes to UNKNOWN (i.e. we were unable to make a
-// determination about our NAT status with enough confidence).
+// EvtLocalRoutability is an event struct to be emitted with the local's node
+// routability changes state.
 //
 // This event is usually emitted by the AutoNAT subsystem.
-type EvtLocalRoutabilityUnknown struct{}
+type EvtLocalRoutability struct {
```

Comment:
maybe `EvtLocalRoutabilityChange`?

Reply:
Yes.

Reply:
`EvtLocalRoutabilityChanged` is in tune with the nomenclature proposed here: https://github.com/libp2p/go-libp2p-core/blob/master/event/doc.go#L8

(by the time the consumer receives this event, it's in the past).